### PR TITLE
Zealery's Hero Section

### PR DESCRIPTION
**Pull request by JulianaTennett**
I created the hero component and built the entirety of the hero on Zealery's page

### CODE REVIEW

File: _includes/components/project__section.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<div class="project__section">
```
Comment (beahuang) [inline _includes/components/project__section.html]: Component name should be project-section

File: _pages/zealery.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
+layout: base
+title: Hello
```
Comment (beahuang) [inline _pages/zealery.html]: Zealery instead of Hello!!!

File: _scss/components/hero.scss [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+  font-family: 'Asap', sans-serif;
+  font-size: 1em;
+  color: #fff;
+}
+
+.hero__img {
+  display: none;
+}
+
+@media (min-width: $bp-tablet-vert) {
```
Comment (beahuang) [inline _scss/components/hero.scss]: move media queries inside of each class

File: _scss/components/hero.scss [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+  .label {
+    display: inline-flex; //sets width to the size of its contents
+    width: auto; //override the width of the mobile size one
+  }
+
+  .label__section {
+    margin-right: 3.5em;
+  }
+
+  .label__section:last-child {
```
Comment (beahuang) [inline _scss/components/hero.scss]: you can do &:last-child inside of .label__section

```
.label__section {
  &:last-child {
  }
}
```

File: _scss/components/hero.scss [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+  .label__section--answer {
+    padding-top: 10px;
+    font-size: .75em;
+  }
+}
+
+
+@media (min-width: $bp-laptop) {
+
+  // .hero__img {
```
Comment (beahuang) [inline _scss/components/hero.scss]: hero__img to hero__img container

File: _includes/components/hero.html [hunk 6]
```diff
@@ -0,0 +1,46 @@
+<div class="hero">
+  <div class="container">
+
+    <!-- hero images-->
+    <div class="hero__img">
```
Comment (beahuang) [inline _includes/components/hero.html]: change this to hero__img-container

File: _includes/components/hero.html [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+      <img class="img-responsive hero__img--left" src="/img/{{include.img1}}" alt="iphone mockup">
```
Comment (beahuang) [inline _includes/components/hero.html]: add hero__img to both hero__img--left and use it for styles that apply on both hero__img

File: _includes/components/project-section.html [hunk 8]
```diff
@@ -0,0 +1,21 @@
+<div class="project-section">
+  <div class="container">
+
+    <h2>{{ include.subhead }}</h2>
+
+    <p>{{ include.par1 }}</p>
+
+    {% if include.img1 != null %}
```
Comment (beahuang) [inline _includes/components/project-section.html]: change to `{% if include.img1 %}

File: _pages/zealery.html [hunk 9]
```diff
@@ -12,19 +12,41 @@
     <!--hero-->
     {% include components/hero.html
     img1 ='zealery/hero--left.png'
-    img2 ='zealery/hero--right.png'
+    img2 = 'zealery/hero--right.png'
```
Comment (beahuang) [inline _pages/zealery.html]: keep the spacing consistent, either `img=''` or `img = ''` everywhere

File: _scss/base/_base.scss [hunk 10]
```diff
@@ -127,8 +127,16 @@ a {
 }
 
 h2 {
+  margin: 0;
+  padding-bottom: 10px;
   font-family: 'Bitter', serif;
```
Comment (beahuang) [inline _scss/base/_base.scss]: define font family as a variable

File: _includes/components/project-section.html (hunk 8 above)
Comment (JulianaTennett) [reply _includes/components/project-section.html]: can you explain why

Comment (JulianaTennett) [reply _includes/components/project-section.html]: and should this be done on every if function?

Comment (beahuang) [reply _includes/components/project-section.html]: basically include.img1 returns null if it doesn't exist so you don't have to check if its != to null.

File: _scss/pages/zealery.scss [hunk 11]
```diff
@@ -0,0 +1,86 @@
...
+      justify-content: flex-end;
+      width: 105vw;
+    }
+  }
+
+  .hero__img {
+    @media (min-width: $bp-tablet-vert) {
+      position: absolute;
+      &--left {
+        position: absolute;
```
Comment (beahuang) [inline _scss/pages/zealery.scss]: duplicate style

File: _scss/pages/zealery.scss [hunk 12]
```diff
@@ -0,0 +1,86 @@
...
+        width: 30vw;
+        top: 400px;
+        right: 20%;
+        @media (min-width: $bp-desktop) {
+          top: 450px;
+          right: 20%;
+        }
+      }
+      &--right {
+        position: absolute;
```
Comment (beahuang) [inline _scss/pages/zealery.scss]: duplicate

File: _scss/pages/zealery.scss [hunk 13]
```diff
@@ -0,0 +1,86 @@
...
+        color: $c-zealery--pink;
+      }
+    }
+  }
+
+  .dark {
+    .project-section {
+      background-color: $c-zealery--pink;
+      h2 {
+        color: #fff;
```
Comment (beahuang) [inline _scss/pages/zealery.scss]: put #fff into a variable

File: _scss/pages/zealery.scss [hunk 14]
```diff
@@ -0,0 +1,86 @@
...
+      p {
+        color: #fff;
+      }
+    }
+  }
+
+  //WIREFRAMES PROJECT SECTION
+  .wireframes {
+    .section--img2 {
+      padding-top: 75px;
```
Comment (beahuang) [inline _scss/pages/zealery.scss]: this is the same as the hifi project section can you consolidate

Comment (JulianaTennett) [reply _scss/pages/zealery.scss]: Haven't finished this, I copy+pasted from wireframes->hifi but the positioning of them will be different in the final iteration

Comment (beahuang) [reply _scss/pages/zealery.scss]: ok!!

File: _includes/components/project-section.html [hunk 15]
```diff
@@ -0,0 +1,21 @@
...
+
+    <h2>{{ include.subhead }}</h2>
+
+    <p>{{ include.par1 }}</p>
+
+    {% if include.img1 %}
+    <img class="img-responsive section--images section--img1" src="/img/{{include.img1}}" alt="{{include.alt}}">
+    {% endif %}
+
+    {% if include.par2 != null %}
```
Comment (beahuang) [inline _includes/components/project-section.html]: here

File: _includes/components/project-section.html [hunk 16]
```diff
@@ -0,0 +1,21 @@
...
+    <p>{{ include.par2 }}</p>
+    {% endif %}
+
+    {% if include.img2 != null %}
```
Comment (beahuang) [inline _includes/components/project-section.html]: here

File: _includes/components/hero.html [hunk 17]
```diff
@@ -0,0 +1,46 @@
...
+    <!-- hero images-->
+    <div class="hero__img--container">
+      <img class="img-responsive hero__img hero__img--left" src="/img/{{ include.img1 }}" alt="iphone mockup">
+      <img class="img-responsive hero__img hero__img--right" src="/img/{{ include.img2 }}" alt="iphone mockup">
+    </div>
+
+    <h1>{{ include.question }}</h1>
+
+    <div class="label">
+      {% if include.role != null %}
```
Comment (beahuang) [inline _includes/components/hero.html]: here

**Pull request merged**